queries.md

```diff
@@ -12,21 +12,41 @@
  - **`projection`**: {name: 1, _id: 0}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
+{name: "Twitter"}
+{name: 1, category_code:1, founded_year:1}
 
 ## 4. Find all the companies who have `web` as their **category_code**, but limit the search to 50 companies.
-
+{category_code: "web"}
+Limit: 50
 ## 5. Find all the companies which **category_code** is 'enterprise' and have been founded in 2005. Retrieve only the `name`, `category_code` and `founded_year` fields.
+{category_code: "enterprise",founded_year: 2005}
+{name:1,category_code:1,founded_year:1}
 
 ## 6. Find all the companies that have been **founded** on the 2000 or have 20 **employees**. Sort them descendingly by their `number_of_employees`.
+{$or: [{founded_year: 2000}, {number_of_employees: 20}]}
+{number_of_employees:-1}
 
 ## 7. Find all the companies that do not include `web` nor `social` on their **category_code**. Limit the search to 20 documents and retrieve only their `name` and `category_code`.
+{$nor: [{category_code: "web"}, {category_code: "social"}]}
+{category_code:1, name:1}
 
 ## 8. Find all the companies that were not **founded** on 'June'. Skip the first 50 results and retrieve only the `founded_month` and `name` fields.
+{founded_month: {$ne: "June"}}
+{founded_month:1, name:1}
+Limit: 50
```

Comment:
nope, it's just Skip, close to limit
![image](https://user-images.githubusercontent.com/29750639/38603681-96a45d90-3d6e-11e8-8168-9b171631e83b.png)
you missed _id:0